hdk-rust/README.md

```diff
@@ -0,0 +1,128 @@
+# Holochain Development Kit for Rust-based Apps
+
+[![Project](https://img.shields.io/badge/project-holochain-blue.svg?style=flat-square)](http://holochain.org/)
+[![PM](https://img.shields.io/badge/pm-waffle-blue.svg?style=flat-square)](https://waffle.io/holochain/org)
+[![Chat](https://img.shields.io/badge/chat-chat%2eholochain%2enet-blue.svg?style=flat-square)](https://chat.holochain.net)
+
+## Overview
+`hdk-rust` is a library for Rust-based holochain dApps that makes it easier to develop Holochain zomes. With Holochain, zome functions and validation code are represented as WASM binaries. This library provides bindings for Rust.
+
+## Usage
+First, [Rust](https://www.rust-lang.org/en-US/install.html) must be installed on your computer.
+
+Being a Rust library, `hdk-rust` can be added as a dependency to any Rust crate. When you generate Rust based Zomes with [holochain-cmd](https://github.com/holochain/holochain-cmd) it will automatically be added as a dependency, and imported into your code.
+
```

Comment:
There are def some updates to make to the readme, but I'll wait till this goes in and propose them